tests/testthat/test-dplyr.R

```diff
@@ -131,6 +131,8 @@ test_that("'across()' works with formula syntax", {
 })
 
 test_that("'across()' emits error when formula references a mutated column", {
+  skip_if_not(dbplyr_uses_ops())
```

Comment:
This works now because it uses dbplyr. Is there any good reason why this should not work?